src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const divElement = document.createElement('div');
+  const heading = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  divElement.classList.add('notification', type);
+  divElement.style.top = `${posTop}px`;
+  divElement.style.right = `${posRight}px`;
+
+  heading.classList.add('title');
+  heading.textContent = title;
+  paragraph.textContent = description;
+
+  divElement.appendChild(heading);
+  divElement.appendChild(paragraph);
+
+  document.body.appendChild(divElement);
+
+  setTimeout(() => {
+    divElement.style.display = 'none';
+  }, 2000);
 };
```

Comment:
It's a good practice to add a function description. This will improve the maintainability of your code and will be helpful for other developers who might work with your code in the future.